### DISCUSSION TITLE
导出 HTML（不含 CSS）用于 SuperMemo 的导入

### DISCUSSION THREAD

**Discussion by Kenshin** [Kenshin/simpread#2475] [插件说明]
> 此功能最低要求 2.2.0 版本，如低于此版本，[请升级](http://ksria.com/simpread/) 到最新版本。

描述
---

> 导出不包含 CSS 的 HTML 文件。

地址
---

[https://simpread.ksria.cn/plugins/details/6FIqWZpXRD](https://simpread.ksria.cn/plugins/details/6FIqWZpXRD)

版本
---

### 1.0.2

  - 导出的纯 HTML，添加 `title` 为 `h1`
  - 导出时的名称修改为 `yyyy-mm-dd_hh-mm-ss-title.html`
  - 修复图片尺寸问题，为原图的 70%

### 0.0.2
  - 修复 `title` 设置错误问题

### 0.0.1
  - 基本功能
  - 支持快捷键 `s m`

位置
---

> 此功能使用了 **插件触发器**，只有在需要时才会调用，而不会随着进入 `阅读模式` 而直接调用。

![image](https://user-images.githubusercontent.com/81074/124066773-dbdd4780-da6b-11eb-9a15-f2f482fe510c.png)

快捷键
---

> 支持快捷键操作，可不用启用 <kbd>s r</kbd> 即可使用，且不可更改。

<kbd>s m</kbd> 

功能
---

> 在阅读模式中，通过点击上图所示 icon 触发**导出 HTML（不含 CSS）** 的功能。

Quicker 动作
---

可配合此 [Quicker](https://github.com/Kenshin/simpread/discussions/2476) 实现 **一键导出**。

注意
---

1. 如果你使用了同步助手，**会引起无法安装扩展的问题**，解决方案 [请看这里](https://github.com/Kenshin/simpread/discussions/2342)。

2. 通过此方式是通过浏览器导出，暂时无法通过 [同步助手](http://ksria.com/simpread/docs/#/Sync) 导出。

引申
---

> 安装插件或带有插件触发器的插件失效的 [解决方案](https://github.com/Kenshin/simpread/discussions/2342)。

**Comment by NNGTHB** [comment]
![image](https://user-images.githubusercontent.com/115760745/210352848-1f738e32-81f5-4809-a1a0-92ffc873b99c.png)
请问能加一个类似这个插件这样的选项吗 https://simpread.ksria.cn/plugins/details/GEyCv9vyXX
导入supermemo后的图片太大了，如果导出的时候能调整就好了

**Reply by Kenshin** [reply]
okay 可以考虑。

**Reply by NNGTHB** [reply]
> okay 可以考虑。

感谢👌
